### PR TITLE
New Port Super Mega Zero

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Super Mega Zero

## Game Information
- **Title**: Super Mega Zero
- **URL**: https://store.steampowered.com/app/1581880/Super_Mega_Zero/

## Submission Requirements

CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] JELOS/ROCKNIX
[X] MuOS

Architecture
[X] armHF
[X] Panfrost (Mainline)

Resolution Tests
Test all major resolutions:
[X] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/supermegazero/port.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Super Mega Zero",
+    "porter": [
+      "Fraxinus88"
+    ],
+    "desc": "Miss Undefined has been whisked away by the dastardly Kill Screen. Go from Zero to Hero in this fast-paced platformer, chasing time, the high score, and eventually the Kill Screen itself.\n\nSuper Mega Zero is a tight precision and puzzle platformer with shoot 'em up elements. Simply touch an upgrade and switch forms to go from classic, straight-forward platforming to the more puzzle-like action number system or Shoot' Em Up gameplay.\n\nNote: Poor readability on small screens and options menu does not work.",
+    "desc_md": null,
+    "inst": "Purchase the game from: https://store.steampowered.com/app/1581880/Super_Mega_Zero/\nRun steam console via Windows+R: Type: steam://open/console. \nDownload the 20 April 2022 version by typing: download_depot 1581880 1581881 8562910234793311857 in the console, when the download is complete, it will tell you the location of the folder,  copy all the files from the download folder to /ports/supermegazero/gamedata",
```
Comment (Cebion) [inline ports/supermegazero/port.json:16]: You have two instruction fields. One for the instructions you see on the device and one inst_md which will be shown on the website. 

It does not make sense to put the full steam link in the Portmaster install instructions on screen. Also you have no formating on the device.

So keep it short on the device instructions or tell them to visit the site for details and the detailed on the inst_md

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/supermegazero/port.json (hunk 1 above)
Comment (Fraxinus88) [reply ports/supermegazero/port.json]: Updated. Kindly review :)

File: ports/supermegazero/port.json [hunk 2]
```diff
@@ -23,8 +23,6 @@
     "exp": false,
     "runtime": null,
     "reqs": [],
-    "arch": [
-      "armhf"
-    ]
+    "arch": []
```
Comment (Cebion) [inline ports/supermegazero/port.json:26]: gmloader is armhf

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/supermegazero/port.json (hunk 2 above)
Comment (Fraxinus88) [reply ports/supermegazero/port.json]: me bads :/

**Pull request merged**